<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QMainWindow" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="lbLogo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>761</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background=rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lbClose">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>5</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 255, 255);
border: 0px solid white;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTracker">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(49, 49, 50)</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;LOTUS TRACKER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 255, 255); 
border: 1px solid gray; 
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btLogin">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>400</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color : rgb(24, 185, 63);
border: 1px solid black;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btNew">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>400</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color : rgb(24, 185, 63);
border: 1px solid black;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>NEW USER</string>
     </property>
    </widget>
    <widget class="QFrame" name="frameLogin">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>271</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray; </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btEnter">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color : rgb(24, 185, 63);
border: 1px solid black;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>ENTER</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>10</y>
        <width>151</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="edLoginEmail">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 255, 255); 
border: 1px solid gray; 
border-radius: 10px;
padding-left: 5px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edLoginPassword">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 255, 255); 
border: 1px solid gray; 
border-radius: 10px;
padding-left: 5px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>10</y>
        <width>62</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="lbForgorPassword">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Forgot password</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frameNew">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>270</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray; </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btRegister">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color : rgb(24, 185, 63);
border: 1px solid black;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>20</y>
        <width>151</width>
        <height>82</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="edNewEmail">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 255, 255); 
border: 1px solid gray; 
border-radius: 10px;
padding-left: 5px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edNewPassword">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 255, 255); 
border: 1px solid gray; 
border-radius: 10px;
padding-left: 5px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edNewConfirm">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 255, 255); 
border: 1px solid gray; 
border-radius: 10px;
padding-left: 5px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhSensitiveData</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>62</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="btBack">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>31</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : rgb(255, 255, 255); 
border: 0px solid gray; </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../resources.qrc">
       <normaloff>:/back.png</normaloff>:/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="lbLoading">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>390</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray; </string>
     </property>
     <property name="text">
      <string>Loading...</string>
     </property>
    </widget>
    <widget class="QFrame" name="frameRecoverPassword">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>280</y>
       <width>281</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray; </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btRecoverPassword">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color : rgb(24, 185, 63);
border: 1px solid black;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>RECOVER PASSWORD</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>96</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="edRecoverPasswordEmail">
         <property name="styleSheet">
          <string notr="true">background-color : rgb(255, 255, 255); 
border: 1px solid gray; 
border-radius: 10px;
padding-left: 5px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhEmailCharactersOnly</set>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>10</y>
        <width>62</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>450</y>
       <width>601</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid gray; </string>
     </property>
     <property name="text">
      <string>Lotus Tracker works without login, but only a logged user will be able to see their deck history, statistics and matches later.</string>
     </property>
    </widget>
    <zorder>frameRecoverPassword</zorder>
    <zorder>frameNew</zorder>
    <zorder>frameLogin</zorder>
    <zorder>btLogin</zorder>
    <zorder>btNew</zorder>
    <zorder>btBack</zorder>
    <zorder>lbLoading</zorder>
    <zorder>label_7</zorder>
   </widget>
   <zorder>frame</zorder>
   <zorder>lbLogo</zorder>
   <zorder>lbTracker</zorder>
   <zorder>lbClose</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
